ulii/views/redirect.py

```diff
@@ -0,0 +1,38 @@
+from django.shortcuts import get_object_or_404
+from django.views.generic import RedirectView
+
+from peachjam.models import Judgment
+
+COURT_CODE_MAPPINGS = {
+    "leadership-code-tribunal-uganda": "UGLCT",
+    "supreme-court-uganda": "UGSC",
+    "court-appeal-uganda": "UGCA",
+    "constitutional-court-uganda": "UGCC",
+    "commercial-court-uganda": "UGCommC",
+    "industrial-court-uganda": "UGIC",
+    "high-court-uganda": "UGHC",
+    "hc-civil-division-uganda": "UGHCCD",
+    "hc-criminal-division-uganda": "UGHCCRD",
+    "hc-family-division-uganda": "UGHCFD",
+    "hc-land-division-uganda": "UGHCLD",
+    "hc-international-crimes-division-uganda": "UGHICD",
+    "hc-anti-corruption-division-uganda": "UGHCACD",
+    "high-court-execution-bailiffs-division-uganda": "UGHCEBD",
+    "election-petitions-uganda": "UGHCEP",
+    "equal-opportunities-commission": "UGEOC",
+    "center-arbitration-dispute-resolution-uganda": "UGCADER",
+    "tax-appeals-tribunal-uganda": "UGTAT",
+}
+
+
+class RedirectCaseURLsView(RedirectView):
+    def get_redirect_url(self, court, year, number, *args, **kwargs):
```

Comment:
I suggest not even checking that the object exists. Simply redirect to the raw URL based on the various parameters. Keeps it simple.

Reply:
This functionality will be used for multiple other legacy LIIs, so maybe we should put it into `liiweb` and have a way of configuring the court mappings from, say, the settings file?

Reply:
> This functionality will be used for multiple other legacy LIIs, so maybe we should put it into `liiweb` and have a way of configuring the court mappings from, say, the settings file?

I had the same thought.